api/static/index.html

```diff
@@ -100,11 +100,15 @@
                 The <span style="color: lightgray">blinking arrows</span> represent electricity <span style="color: lightgray">imports and exports</span>.<br />
                 <br />
                 <span style="color: lightgray; font-size: 14px; color: white;"><i>Tip: Click on a country to start exploring ⟶</i></span><br />
+		<br />
                 <div class="country-picker-container" style="text-align: center">
                     <select class="country-picker" onchange="onSelectedCountryChange();"></select>
                 </div>
             </div>
-            
+	    <svg class="country-table" style="display: none"></svg>
```

Comment:
That is not supposed to be there. But actually, I wanted to include the same line that separates the heading from table content at the bottom, after the TIP, for beter visuals. I found that the <svg> was for that but it does not seem to work inside a table. A line (<hr>) on the other hand is too thick and looks different than the one on top so its not ok to include that. Should there be any line at all though?